scada-lts-e2e-service/scada-lts-e2e-service-impl/pom.xml

```diff
@@ -17,5 +17,9 @@
             <groupId>org.scadalts</groupId>
             <artifactId>scada-lts-e2e-service-core</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.scadalts</groupId>
+            <artifactId>scada-lts-e2e-page-impl</artifactId>
+        </dependency>
```

Comment:
It may stay that way for now. We will have to move the Criteria / Dict implementations to Common.